lib/ansible/plugins/become/__init__.py

```diff
@@ -77,12 +77,23 @@ def _check_password_error(self, b_out, msg):
         return b_fail and b_fail in b_out
 
     def check_incorrect_password(self, b_output):
+        try:
+            self.fail = self.get_option('become_fail_match')
```

Comment:
Should we allow regex here? Similar to what we do in doas and su for password prompts?

Reply:
guess that would be most flexible

Reply:
but we already use dgettext to try to get 'untranslated', not sure this will mesh well